site/themes/dojo/layout/_partial/landingPage/upAndRunning.ejs

```diff
@@ -110,13 +110,19 @@
 
 			requestAnimationFrame(animate);
 		}
-
+		
 		var typing1Elem = document.querySelector('#upAndRunning-typing-1 .typed-text');
+		var fontSize = window.getComputedStyle(typing1Elem, null).getPropertyValue('font-size');
+
+		window.onresize = function() {
```

Comment:
If this function was set to a local variable, it could be called above to set `fontSize` and would reduce duplication.

Reply:
:+1: